plugins/org.jkiss.dbeaver.registry/src/org/jkiss/dbeaver/registry/DataSourceSerializerModern.java

```diff
@@ -829,7 +829,7 @@ private Map<String, Map<String, Map<String, String>>> readSecureCredentials(
                 }
             }
             log.error("Error reading secure credentials", e);
-            throw new DBException("Project configuration can not be open", e);
+            return null;
```

Comment:
I'm not sure about this change. Haven't we have the case when changing error to null broke something?

Reply:
Sure it looks a bit strange but we have to return null in order to proceed open datasources (requirement) with lost credentials

Reply:
Im rethinking a bit of approach, added  specific condition.